modules/age.nix

```diff
@@ -187,6 +187,13 @@ in {
         The age executable to use.
       '';
     };
+    pluginPackages = mkOption {
```

Comment:
I was thinking perhaps it would be nicer to name this `path`, similar to `systemd.services.<name>.path`. On the other hand, the name `pluginPackages` communicates more intent and I believe is more discoverable. Thoughts? :)